src/backend/utils.ts

```diff
@@ -712,11 +713,24 @@ function detectVCRedist(mainWindow: BrowserWindow) {
         })
       }
     } else {
-      logInfo('VCRuntime is installed', LogPrefix.Backend)
+      logInfo('VCRuntime is installed', { prefix: LogPrefix.Backend })
     }
   })
 }
 
+export function notify({ body, title }: NotifyType) {
+  if (Notification.isSupported() && !isSteamDeckGameMode) {
+    const mainWindow = BrowserWindow.getAllWindows()[0]
+    const notify = new Notification({
+      body,
+      title
+    })
+
+    notify.on('click', () => mainWindow.show())
+    notify.show()
+  }
+}
+
```

Comment:
Why was this moved?

Reply:
I think somehow the function was implemented twice. I removed one. I can move the function to the end again if you want. But actually does not hurt to let it there. The implementation didn't change.

Reply:
Yeah that's fine, was just wondering why it moved